Firestore/core/src/firebase/firestore/remote/datastore.h

```diff
@@ -29,19 +40,45 @@ namespace remote {
 
 class Datastore {
  public:
+  Datastore(util::AsyncQueue* firestore_queue,
```

Comment:
Please reuse the variables names we use on the other platforms (or change them everywhere). This should be the worker_queue.

Firestore has multiple queues: we have the worker queue and the user callback queue, so "firestore queue" is actually ambiguous as a name.